lib/collector/remote-method.js

```diff
@@ -336,13 +336,14 @@ RemoteMethod.prototype._userAgent = function _userAgent() {
 /**
  * Generate a URL the collector understands.
  *
+ * @param {boolean} redactLicenseKey flag to redact license key in path
  * @returns {string} The URL path to be POSTed to.
  */
-RemoteMethod.prototype._path = function _path() {
+RemoteMethod.prototype._path = function _path(redactLicenseKey) {
   const query = {
     marshal_format: 'json',
     protocol_version: this._protocolVersion,
-    license_key: this._config.license_key,
+    license_key: redactLicenseKey ? 'REDACTED' : this._config.license_key,
```

Comment:
Do we want to redact entirely, or just partially?

Reply:
i don't see any value in having partial

Reply:
The only one I can think of is to get the first few characters, if we're trying to verify if they're using the wrong key...but a 403 response would be enough to show that.